src/ddo/NvmConfig.ts

```diff
@@ -44,6 +44,13 @@ export interface NvmConfig {
     versions: NvmConfigVersions[]
 }
 
+export enum ImmutableBackendsSupported {
```

Comment:
I think this could be changed to just `ImmutableBackends`

Reply:
done